content/release-notes/1.12.0.md

```diff
@@ -0,0 +1,38 @@
+---
+date: 2020-01-31T19:55:39-08:00
+linktitle: "1.12.0"
+title: "1.12.0"
+---
+
+{{<features>}}
+* Namespace can be specified for Helm charts.
+* kurl.sh supports CentOS 7.7
+* Node resource analyzers
+{{</features>}}
+
+{{<changes>}}
+* If license is updated during a sync, a dialog is shown linking to version history.
+* Better feedback when saving app config.
+* Support bundle analysis now has breadcrumbs.
+* `replicated/kurl-util` container uses a versioned tag instead of `latest`.
+* `kots` CLI now includes an `Authorization` header when making requests to `kotsadm-api`.
+* Airgap upload endpoints now require `Authorization` headers.
+* Helm chart rendering errors are now displayed in the Admin Console.
+* The Dashboard link has been removed from the license upload pages.
+* License is synced to the latest version on installation.
+* `kotsadm-api` no longer needs to be restarted when console password is reset.
```

Comment:
Sounds like a bug fix. So when we use the CLI to reset password, it didn't take until we also restart kotsadm-api?